CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# Contributing to Theia C/C++ Extensions
+
+This repository is the home to Theia's C/C++ extensions.
+
+See Theia's main repository to get more information about the IDE:
+https://github.com/theia-ide/theia
```

Comment:
replace "theia-ide" with "eclipse"? Will need to update it soon anyway if not now.